text/0000-new-browser-support-policy.md

```diff
@@ -0,0 +1,67 @@
+- Start Date: 2020-05-21
+- Relevant Team(s): All
+- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+- Tracking: (leave this empty)
+
+# New Browser Support Policy
+
+## Summary
+
+> In dropping support for IE11 at some point in the future, we will need a new
+browser support policy. This is the proposed policy.
+
+## Motivation
+
+> It should be clear which browsers are supported by Ember.js
+
+## Detailed policy
+
+- Desktop
+
+    1. Google Chrome
+
+        - The current and previous stable releases are supported
```

Comment:
Idk if this is enough. I know of companies that have automatic updates turned off and are on chrome 65 still

Reply:
Agreed. I wish Chrome had something similar to Firefox's ESR. What do you suggest?

Reply:
How about the same as iOS. All versions of chrome released in the past 36 months? (could be the same for FF, Safari too). So no longer tied to version but to release date.